client/src/components/Form/FormListElementOperations.vue

```diff
@@ -0,0 +1,70 @@
+<script setup lang="ts">
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faCaretDown, faCaretUp, faTrashAlt } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import { BButton, BButtonGroup } from "bootstrap-vue";
+
+// @ts-ignore: bad library types
+library.add(faTrashAlt, faCaretUp, faCaretDown);
```

Comment:
We figured out some time ago that we no longer need to add icons to the library to be properly bundled if we bind the icon definition directly instead of the "string representation", which is cleaner and easier to grep when looking for the icon usage.

```suggestion
import { faCaretDown, faCaretUp, faTrashAlt } from "@fortawesome/free-solid-svg-icons";
import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
import { BButton, BButtonGroup } from "bootstrap-vue";

```